src/tracer.js

```diff
@@ -316,8 +316,14 @@ function sendCurrentTrace(traceSender) {
         platform: tracerObj.trace.getPlatform(),
     };
 
-
-    if (JSON.stringify(traceJson).length >= consts.MAX_TRACE_SIZE_BYTES) {
+    let stringifyTraceJson;
+    try {
+        stringifyTraceJson = JSON.stringify(traceJson);
+    } catch (err) {
+        utils.debugWarn('Skip send traces.', err);
```

Comment:
can you change this to:
```
Epsagon - no trace was sent since there was an error serializing the trace. Please contact support.
```

Reply:
I meant to actually use `console.warn` here (not just if `EPSAGON_DEBUG` is true) - if this happens the user should be alerted so he can contact us